### PR TITLE
Create package for consul-k8s integration

### PR DESCRIPTION
**Pull request by stevendborrelli**

### CODE REVIEW

File: .bintray [hunk 1]
```diff
@@ -1,2 +1,2 @@
-BINTRAY_PROJECT=shippedrepos/mantl-1.2
+BINTRAY_PROJECT=shippedrepos/mantl-1.2-k8s
```
Comment (BrianHicks) [inline .bintray]: this will change the bintray project for all the packages. Is that what you want?

File: consul/consul-k8s/consul.service [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+[Service]
+User=consul
+Group=consul
+EnvironmentFile=/etc/sysconfig/consul
+ExecStart=/usr/bin/consul agent -config-dir=${CONFIG_DIR} -data-dir=${DATA_DIR} $EXTRA_OPTS
+ExecReload=-/bin/kill -HUP $MAINPID
+KillSignal=SIGINT
+LimitNOFILE=65536
+Restart=on-success
+IgnoreSIGPIPE=yes
```
Comment (siddharthist) [inline consul/consul-k8s/consul.service]: What does this do?

File: .bintray (hunk 1 above)
Comment (stevendborrelli) [inline .bintray]: I'll change it to the main repo.

File: consul/consul-k8s/consul.sysconfig [hunk 3]
```diff
@@ -0,0 +1,3 @@
+CONFIG_DIR=/etc/consul
```
Comment (siddharthist) [inline consul/consul-k8s/consul.sysconfig]: Why have this in a separate directory?

File: consul/consul-k8s/consul.service (hunk 2 above)
Comment (BrianHicks) [inline consul/consul-k8s/consul.service]: It's something to do with systemd trying to write to closed pipes and nuking processes. Real weird error, but that fixed it.

File: consul/consul-k8s/consul.sysconfig (hunk 3 above)
Comment (BrianHicks) [inline consul/consul-k8s/consul.sysconfig]: The consul directory contains healthchecks, etc. It needs its own directory.

Comment (siddharthist) [inline consul/consul-k8s/consul.sysconfig]: Oh, I should've said "in a separate file"? Why not have it defined in the systemd unit directly?

Comment (BrianHicks) [inline consul/consul-k8s/consul.sysconfig]: Same reason we use variables for anything else: so you can change it without invoking `daemon-reload`.

Comment (siddharthist) [inline consul/consul-k8s/consul.sysconfig]: Gotcha, thanks!

**Comment by BrianHicks** [comment]
@stevendborrelli LGTM

**Pull request merged**